src/macros.jl

```diff
@@ -937,7 +931,7 @@ macro variable(args...)
         anon_singleton = true
     else
         x = shift!(extra)
-        if x in [:Int,:Bin,:SDP]
+        if x in [:Int,:Bin,:PSD]
```

Comment:
I've been ignoring deprecations so far, there are too many changes without 1-1 replacements and we still have unsettled syntax questions. I think we'll have to do a deprecation pass later on. We can write the upgrading guide at the same time.